#! /bin/sh /usr/share/dpatch/dpatch-run
## 06_strdup_implicit_pointer_conversion.dpatch by OHURA Makoto <ohura@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix Bug#464465
## DP: Added -D_GNU_SOURCE to CFLAGS to enable prototype definition
## DP: of strdup(3).

@DPATCH@
diff -urNad vflib3-3.6.14.dfsg~/configure.in vflib3-3.6.14.dfsg/configure.in
--- vflib3-3.6.14.dfsg~/configure.in	2008-02-09 00:02:49.000000000 +0900
+++ vflib3-3.6.14.dfsg/configure.in	2008-02-09 00:10:21.000000000 +0900
@@ -46,7 +46,7 @@
 AC_PROG_LN_S
 
 if test "x$CC" = xgcc; then
-	XX_CFLAGS="-g -Wall -pedantic -ansi"
+	XX_CFLAGS="-g -Wall -pedantic -ansi -D_GNU_SOURCE"
 else
        	XX_CFLAGS=
 fi
